### PR TITLE
Destination weather api

### PR DESCRIPTION
**Pull request by eifinger**
Add missing methods for destination weather api

### CODE REVIEW

File: herepy/destination_weather_api.py [hunk 1]
```diff
@@ -12,13 +12,11 @@
...
 
+
 class DestinationWeatherApi(HEREApi):
     """A python interface into the HERE Destination Weather API"""
 
-    def __init__(self,
-                 app_id=None,
-                 app_code=None,
-                 timeout=None):
+    def __init__(self, app_id=None, app_code=None, timeout=None):
```
Comment (abdullahselek) [inline herepy/destination_weather_api.py]: I guess keeping new lines is better for readibility.

**Comment by eifinger** [comment]
The files are formatted with black: https://github.com/psf/black

I can still reformat if you like.

**Comment by abdullahselek** [comment]
I totally forgot that you use black all good than good to go.

**Comment by eifinger** [comment]
@abdullahselek could you please release a new version? Upcoming homeassistant component 😉

**Pull request merged**